backend/model_server/encoders.py

```diff
@@ -199,7 +199,10 @@ def get_embedding_model(
 
     if model_name not in _GLOBAL_MODELS_DICT:
         logger.info(f"Loading {model_name}")
-        model = SentenceTransformer(model_name)
+        model = SentenceTransformer(
+            model_name_or_path=model_name,
+            trust_remote_code=True,
```

Comment:
I would add a comment here:
"Some model architectures that aren't built into the Transformers or Sentence Transformer need to be downloaded to be loaded locally. This does not mean data is sent to remote servers for inference, however the remote code can be fairly arbitrary so only use trusted models"